v3-docs/docs/api/accounts.md

```diff
@@ -773,10 +773,45 @@ sign-in process, it also supports email-based sign-in including
 address verification and password recovery emails.
 
 The Meteor server stores passwords using the
-[bcrypt](http://en.wikipedia.org/wiki/Bcrypt) algorithm. This helps
+[argon2](http://en.wikipedia.org/wiki/Argon2) algorithm. This helps
 protect against embarrassing password leaks if the server's database is
 compromised.
 
+### Password encryption and security
+
+Passwords are hashed on the client using **SHA-256** algorithm before being sent to the server. This ensures that sensitive data is never transmitted in plain text. Once received by the server, the hashed value is further encrypted and securely stored in the `Meteor.users` collection.
+
+Starting from `accounts-passwords:4.0.0`, the encryption algorithm used is [Argon2](http://en.wikipedia.org/wiki/Argon2). This algorithm is specifically designed to resist GPU-based brute force attacks. Argon2 has replaced the previous [bcrypt](http://en.wikipedia.org/wiki/Bcrypt) algorithm as it's now regarded as a more robust option. For more details, see the [OWASP Password Storage Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html).
+
+**Migrating from `bcrypt` to `argon2`**
+
+The transition from `bcrypt` to `argon2` is handled seamlessly during user logins. When a user logs in, their password is first verified against the existing bcrypt hash. If successful, the password is re-encrypted using Argon2 and the new hash is stored in the database.
+
+**Configuring `argon2` parameters**
+
+The accounts-password package allows limited customization of the Argon2 algorithm's parameters. The configurable options include:
+
+- `timeCost` (default: 3) – This controls the computational cost of the hashing process, affecting both the security level and performance.
+
+To update the timeCost, use the following configuration:
+
+```js
+Accounts.config({
+    argon2Iterations: 4
+});
+```
+
+Other Argon2 parameters, such as `hashLength`, `memoryCost`, `parallelism`, and `type`, are set to default values:
```

Comment:
We should probably offer config API to set them as well.

Reply:
Following up here. I propose to provide `type`, `timeCost`, `memoryCost` and `parallelism`  as config options.

HashLength is not configurable because it is not stored in the argon2 hash, hence not enabling a seamless change of config without storing it in database in an extra field.